stdlib/2and3/datetime.pyi

```diff
@@ -1,31 +1,37 @@
 import sys
 from time import struct_time
-from typing import Optional, SupportsAbs, Tuple, overload
+from typing import AnyStr, Optional, SupportsAbs, Tuple, Union, overload, \
+    ClassVar
```

Comment:
Could you please use ``import (...)`` instead of ``\``? pytype doesn't like it, and it is also the preferred style in mypy.